vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -50,11 +51,24 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
       [props = context->Properties()]() { return Microsoft::React::CreateHttpModule(props); },
       jsMessageQueue);
 
+  if (!Microsoft::React::GetRuntimeOptionBool("Http.UseMonolithicModule")) {
+    modules.emplace_back(
+        Microsoft::React::GetBlobModuleName(),
+        [props = context->Properties()]() { return Microsoft::React::CreateBlobModule(props); },
+        jsMessageQueue);
+
+    modules.emplace_back(
+        Microsoft::React::GetFileReaderModuleName(),
+        [props = context->Properties()]() { return Microsoft::React::CreateFileReaderModule(props); },
+        jsMessageQueue);
+  }
+
   modules.emplace_back(
       "Timing",
       [batchingUIMessageQueue]() { return facebook::react::CreateTimingModule(batchingUIMessageQueue); },
       batchingUIMessageQueue);
 
+  // TODO: For reviewers - Why does this factory move the context object?
```

Comment:
For reviewers.

Reply:
Error prone indeed. I was getting crashes during my testing.

I'll add a comment to explain why this is done.